src/hbbft.erl

```diff
@@ -443,3 +445,11 @@ serialize_hbbft_data(#hbbft_data{batch_size=BatchSize,
 -spec is_serialized(hbbft_data() | hbbft_serialized_data()) -> boolean().
 is_serialized(Data) when is_record(Data, hbbft_serialized_data) -> true;
 is_serialized(Data) when is_record(Data, hbbft_data) -> false.
+
+group_by(Tuples) ->
```

Comment:
Nitpicking here, move this to hbfft_utils perhaps?

Reply:
I'm hesitant to put functions with a single callsite in a utility module. If we need it elsewhere we should move it then.